src/entity_utils.cc

```diff
@@ -227,20 +228,19 @@ spawn_floating_generic_entity(glm::mat4 mat, const std::string &mesh, const std:
     // todo: fix this
     *type.name = "Generic";
 
-    pop_entity_off(ce);
+    pop_entity_off(ce, face);
 
     return ce;
 }
 
-void pop_entity_off(c_entity entity) {
+void pop_entity_off(c_entity entity, surface_index surface /*= surface_index_none*/) {
```

Comment:
What is this surface_index_none?

Reply:
hmm, I may not need that anymore. It was used previously for applying a force straight out along normal, and wasn't need for non-embedded surface meshes. Since I'm not allowing them to be embedded, and the code using this is probably gone, I should remove it.

Thanks

Reply:
You don't seem to use `surface` in this function at all now.

Reply:
done